android/src/main/java/com/reactnativestripesdk/PaymentMethodCreateParamsFactory.kt

```diff
@@ -5,11 +5,12 @@ import com.stripe.android.model.*
 
 class PaymentMethodCreateParamsFactory(
   private val clientSecret: String,
-  private val params: ReadableMap,
+  private val paymentMethodData: ReadableMap?,
+  private val options: ReadableMap,
```

Comment:
Just a quick clarification to make sure I understand the change. Previously params could contain the paymentMethodData or options. Now we are being explicit to include paymentMethodData and what was params now becomes options?